tests/core/test_invite.py

```diff
@@ -0,0 +1,292 @@
+# Parsec Cloud (https://parsec.cloud) Copyright (c) AGPLv3 2019 Scille SAS
+
+import pytest
+import trio
+
+from parsec.api.protocol import DeviceID, DeviceName, HumanHandle, HandshakeInvitedOperation
+from parsec.backend.invite import DeviceInvitation, UserInvitation
+from parsec.core.backend_connection import (
+    backend_invited_cmds_factory,
+    backend_authenticated_cmds_factory,
+)
+from parsec.core.types import BackendInvitationAddr, LocalDevice
+from parsec.core.invite import (
+    InvitePeerResetError,
+    claimer_retreive_info,
```

Comment:
"retrieve" in English :p

Reply:
7e537da6bdf1a4518b820394b1bfd7a4d3f4a177 -__-'''